nrf_desktop: Remove application configuration with SW_SPLIT LL (#5763)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Added documentation for :ref:`nrf_desktop_usb_state_pm`.

* Removed:

>   * Removed configuration files for building the application with SW_SPLIT LL for various boards.

Reviewer: I might be wrong, but isn't this :kconfig:`SW_SPLIT_LL`?

Author: its :kconfig:`BT_LL_SW_SPLIT` it wasn't my intention to put the exact name but why not. I will push in a minute.

>   * Removed configuration files for building the application with :kconfig:`BT_LL_SW_SPLIT` for various boards.

Author: done

>     The boards that can not fit SoftDevice LL due to Flash/SRAM limitations remains untouched.

Reviewer: ```suggestion
    The boards that can not fit SoftDevice LL due to flash or SRAM limitations remain untouched.
```

>   * Removed configuration files for building the application with :kconfig:`CONFIG_BT_LL_SW_SPLIT` for various boards.
>     The boards that can not fit SoftDevice LL due to flash or SRAM limitations remain untouched.

Reviewer: ```suggestion
  * Removed configuration files used for building the application with :kconfig:`CONFIG_BT_LL_SW_SPLIT` for various boards.
    The configuration files for boards that do not have room for the SoftDevice LL in flash or SRAM remain untouched.
```
